new/LIT6216MGAwheyos.xml

```diff
@@ -0,0 +1,103 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6216MGAwheyos" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Miracle of St George: St George punishes to death
+                    governor ʾAwhǝyos after his attempt to destroy the saint’s church in Joppe </title>
+                <title xml:id="t2" xml:lang="en">How Awhǝyos, sent by Diocletian to build pagan
```

Comment:
ʾAwhǝyos with initial ʾ

Reply:
I copied the title as it is from Strelcyn; I also did not adjust the writing of Awḫeyōs by Grebaut/Tisserant. Should we standardize those readings?